rts/lra/client/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder

```diff
@@ -1,2 +1 @@
 org.jboss.resteasy.client.jaxrs.internal.ResteasyClientBuilderImpl
-org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl
```

Comment:
Do we know if it might be appropriate to remove `org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl` from https://github.com/jbosstm/narayana/blob/e6f02e25950677ab06493839997447248e98c408/rts/lra/jaxrs/src/main/resources/META-INF/services/javax.ws.rs.client.ClientBuilder too?